src/components/shared/MintButton.tsx

```diff
@@ -74,15 +75,21 @@ const MintButtonComponent: React.StatelessComponent<Props> = props => {
       />
       <Text style={{ marginTop: 4 }}>
         <MixedText
-          style={[fontSizes.small]}
+          style={[fontSizes.small, { textAlign: "center" }]}
           content={[
-            "The current mint rate is",
+            "Create ",
             {
               currencyType: CurrencyType.Raha,
-              value: new Big(RAHA_UBI_WEEKLY_RATE),
+              value: new Big(RAHA_MINT_WEEKLY_RATE),
               role: CurrencyRole.None
             },
-            "per week."
+            "a week, limit",
+            {
+              currencyType: CurrencyType.Raha,
+              value: new Big(RAHA_MINT_MAX),
+              role: CurrencyRole.None
+            },
+            "per MINT."
```

Comment:
all caps mint is weird. let's make it

"You may create/mint R10.00 per week up to a maximum of R40.00 at once."

Or maybe,

Basic Income Rate: R10.00 per week
Max outstanding basic income: R40.00